docs/getting-started.md

```diff
@@ -0,0 +1,142 @@
+
+Once the process is completed, you should have a new Glee app ready for development and see these files that were made.
+
+![glee_structure](/assets/glee_struct.png)
+
+#### Define our Spec for our API
+
+Glee being a spec-first framework, development starts with defining your API spec. To know more details into it, you can follow glee template to understand it step by step. For our case we will define our API:
+
+```yaml
+asyncapi: 2.1.0
+info:
+  title: Greet Bot
+  version: 0.1.0
+servers:
+  websockets:
+    url: ws://0.0.0.0:3000
+    protocol: ws
+channels:
+  greet:
+    publish:
+      operationId: onGreet
+      message:
+        $ref: '#/components/messages/time'
+    subscribe:
+      message:
+        $ref: '#/components/messages/greet'
+components:
+  messages:
+    time:
+      payload:
+        type: object
+        properties:
+          currentTime:
+            type: number
+          name:
+            type: string
+    greet:
+      payload:
+        type: string
+
+```
+
+This will be the Specification that defines our API, in our case, it is very simple, as we will be sending a name and the time of the day, and our API will greet us accordingly.
+
+One thing to note here is the `operationId`, this is needed and is a crucial part of glee, as this is how we will be connecting our business logic with our spec, `operationId` is the name of the function that will be called every time a certain operation occurs. In our case whenever `/greet` channel received a message.
+
+#### Define our operation function
+
+Now for our case, we will be adding a file `functions/onGreet.js` and writing up the logic for parsing our time and sending a response.
+
+```javascript
+export default async function (event) {
+  const { name, time } = event.payload
+  const t = new Date(time)
+  const curHr = t.getHours()
+  let response = ''
+  if (curHr < 12) {
+    response = `Good Morning ${name}`
+  } else if (curHr < 18) {
+    response = `Good Afternoon ${name}`
+  } else {
+    response = `Good Evening ${name}`
+  }
+  return {
+    reply: [
+      {
+        payload: response,
+      },
+    ],
+  }
+}
+
+```
+
+Every file in the functions folder acts as a handler to develop business logic for glee, every file should export an async function that receives an event parameter, where you have access to payload and server details.
+
+Running and testing your application
+We will not execute the application and carry out testing with Postman to ensure that it is functioning as intended.
+
+Now to run your glee application, just run:
+
+```
+npm run dev
+# or 
+npm run start
+```
+To send a WebSocket request with a payload e.g. `{"name":"souvik", "time": "1567906535"}` to `ws://localhost:3000/greet`, open Postman and checkout the endpoint:
```

Comment:
```suggestion
To send a WebSocket request with a payload e.g. `{"name":"SomeOtherName", "time": "1567906535"}` to `ws://localhost:3000/greet`, open Postman and checkout the endpoint:
```

Everything else is fine, just remove my name I think you followed my example which is good, but remove my name 🤣. Remove it from the image as well, please.